src/Compilers/CSharp/Test/Emit2/Semantics/PatternMatchingTests.cs

```diff
@@ -8108,63 +8108,68 @@ static void Test(ReadOnlySpan<char> chars)
 or: False
 and: False
 not: True")
-                .VerifyIL("C.Test", @"
+                .VerifyIL("C.Test", """
 {
-  // Code size      161 (0xa1)
+  // Code size      167 (0xa7)
```

Comment:
The reason for this change is that we no longer choose the (inverted) linear rewriter for `or` patterns which assigns a flag.